src/main/kotlin/com/team4099/robot2020/subsystems/Shooter.kt

```diff
@@ -0,0 +1,88 @@
+import com.revrobotics.ControlType
+import com.team4099.lib.logging.HelixLogger
+import com.team4099.lib.motorcontroller.SparkMaxControllerFactory
+import com.team4099.lib.subsystem.Subsystem
+import com.team4099.robot2020.config.Constants
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard
+import java.lang.Math.abs
+
+object Shooter : Subsystem {
+    private val masterSparkMax = SparkMaxControllerFactory.createDefaultSparkMax(
+            Constants.Shooter.MASTER_SPARKMAX_ID)
+    private val slaveSparkMax = SparkMaxControllerFactory.createPermanentSlaveSparkMax(
+            Constants.Shooter.SLAVE_SPARKMAX_ID, masterSparkMax)
+
+    enum class State {
+        SHOOTING, IDLE, ACCELERATING
```

Comment:
not sure if shooting is a good name for this state but not sure what else to call it

Reply:
remove accelerating